Individual Project Milestone 1

jon-bell opened the pull request.

jon-bell committed:
wip

jon-bell committed:
Add draft of IP1

mwand commented:
twon => town
Should style guide include naming conventions for files?   camelCase.ts ?  ICamelCase?   
TODO: Partial marks?

jon-bell committed:
Finalize IP D1

jon-bell commented:
> twon => town

Fixed, thanks

> Should style guide include naming conventions for files? camelCase.ts ? ICamelCase?

Students are not creating any new files for this assignment. The course style guide does not currently take a position on this - I would be OK with implementing such a rule, but it would need to be checked by eslint, and any existing files would need to be patched to be conformant.

> TODO: Partial marks?

Now added
